### PR TITLE
test(example): add auth and login-page tests

### PR DESCRIPTION
**Pull request by bcabanes**
Hi!

This PR adds some tests for the example application.

Feel free to ask any change, available to do so!

### CODE REVIEW

File: example-app/app/auth/reducers/auth.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      const createAction = new LoginSuccess({ user });
+      const expectedResult = {
+        loggedIn: true,
+        user: { name: 'test' },
+      };
+      const result = reducer(fromAuth.initialState, createAction);
+      expect(result).toEqual(expectedResult);
+    });
+
+    it('should NOT authenticate a user', () => {
```
Comment (brandonroberts) [inline example-app/app/auth/reducers/auth.spec.ts]: Should this test be somewhere else?

File: example-app/app/auth/reducers/auth.spec.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      const action = {} as any;
+
+      const result = reducer(undefined, action);
+      expect(result).toEqual(fromAuth.initialState);
+    });
+  });
+
+  describe('LOGIN_SUCCESS', () => {
+    it('should login a user', () => {
+      const user = { name: 'test' } as User;
```
Comment (brandonroberts) [inline example-app/app/auth/reducers/auth.spec.ts]: Preference, separate the arrange, act and asset sections of the test with new lines.

File: example-app/app/auth/reducers/login-page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        pending: false,
+      };
+
+      const result = reducer(fromLoginPage.initialState, createAction);
+      expect(result).toEqual(expectedResult);
+    });
+  });
+
+  describe('LOGIN_FAILURE', () => {
+    it('should have no error and no pending state', () => {
```
Comment (brandonroberts) [inline example-app/app/auth/reducers/login-page.spec.ts]: Should have an error

File: example-app/app/auth/reducers/auth.spec.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    it('should return the default state', () => {
+      const action = {} as any;
+
+      const result = reducer(undefined, action);
+      expect(result).toEqual(fromAuth.initialState);
+    });
+  });
+
+  describe('LOGIN_SUCCESS', () => {
+    it('should login a user', () => {
```
Comment (brandonroberts) [inline example-app/app/auth/reducers/auth.spec.ts]: Be more descriptive about what's happening.

**Comment by bcabanes** [comment]
Changes done! 👍

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**